### PR TITLE
/around - responsive

### PR DESCRIPTION
**Pull request by fuentesloic**

### CODE REVIEW

**Review by vinyll — COMMENTED** [review]
On a iPad the top padding of a card is broken:
screen shot 2017-05-25 at 8 25 12 am

On a mobile, it's quite unusable: there's no map but cards, and when clicking on a card, other cards disappear (I understand the technical reason, but it's unusable).

I would say to fallback displaying the map on mobile.
We could have the map on the upper half of the screen, and the cards on the lower half of the screen. When scrolling the cards, the map stays in place.
What do you think?

If that's too complex, I'd say we just display on a mobile like we display on a 600px… :/

**Comment by fuentesloic** [comment]
that was not a problem of complexity more a probleme of my imagination 'cause a tried a lot of way without find out the good one. Thats sounds good for me so ... let's go :)

**Comment by fuentesloic** [comment]
Large screen: pois card into two column
Middle screen: pois card into one column
Tablet screen: pois card into one column without all the information (comments is too huge)
Mobile screen: not pois card replacing by popup directly on the marker

File: static/js/around-map.js [hunk 1]
```diff
@@ -70,7 +81,7 @@ map.on('click', event => {
 // functions
 function currentMarker (currentLatLng) {
   const popup = new mapboxgl.Popup({offset: [10, 0]})
-    .setText('Your current location')
+    .setHTML('<p>Your current location</p>')
```
Comment (vinyll) [inline static/js/around-map.js]: I would just say "<p>My current location</p>" instead, as we are using My/I/me on the project.

**Review by vinyll — COMMENTED** [review]
Why 478px? Why not just 500px?

File: static/js/around-map.js (hunk 1 above)
Comment (fuentesloic) [reply static/js/around-map.js]: sure ... I was pretty sure to change that a long time ago :)

**Comment by fuentesloic** [comment]
why not :p ? (iphone 6+ size so a big screen for a mobile)

**Comment by vinyll** [comment]
> why not :p ? (iphone 6+ size so a big screen for a mobile)

Because it seems more like a random value than an actual break-the-page layout.
I guess you should set a limit break by checking until what value the blocks still look nice, and rounding it up. @newick may have a better opinion about that.

**Comment by fuentesloic** [comment]
> @newick may have a better opinion about that.

I agree with you ! @newick  heeeelp :)

**Review by vinyll — APPROVED** [review]
We really need to clarify the size of responsiveness.
I've been looking over the web and could eventually find this: http://www.websitedimensions.com

However, I once had read the concept of displaying each module (component-wise) until it has the appropriate space. Meaning that each module could have its responsive size, and I thought it made much more sense.

Nevertheless, I don't want to block this PR because of this, and I'd rather have @newick feedback than my non-css expertise.

**Comment by fuentesloic** [comment]
I 100% agree !

**Comment by newick** [comment]
@fuentesloic @vinyll 
Usually I use Bootstrap mediaqueries values 🙄
I tend to change that by a value defined by the design. If it starts to break at a random value, it could be the MQ value.

It seems to be the same idea as the last @vinyll 's one. 
I'll try to find some time this week to look after this problem

**Comment by vinyll** [comment]
@newick: do you still want to have a look at this?

**Comment by vinyll** [comment]
@fuentesloic merge?

**Comment by newick** [comment]
sorry I didn't take time to rewrite my deleted comment!

It looks ok for me, juste instead of 70/30 for tablets i'll stay at 50/50.
There's a bug under something like 500-ish, the POI don't have height anymore

**Comment by fuentesloic** [comment]
I want keep the 70/30 for tablets because I don't have enough space to write `comments` so I prefer have a little card with few important informations and a big map to navigate than cards with a bigger space available and a smallest card (that don't bring to users more informations)
But we could change this way/mind and discuss deepper about it after (like a PR with design/user XP subject).

**Comment by fuentesloic** [comment]
@newick  I fixed the height ;)

**Pull request merged**